src/main/java/sirius/tagliatelle/compiler/Compiler.java

```diff
@@ -518,12 +518,20 @@ public Expression parseExpression(boolean skipWhitespaces) {
      */
     private String consumeStaticBlock() {
         StringBuilder sb = new StringBuilder();
+        Integer numberOfOpenBlocks = 1;
```

Comment:
use `int` instead of `Integer` to prevent autoboxing/unboxing at `++`, `--` and `== 0`